File: introduction/ITP1_3_D-How Many Divisors?.java
import java.util.*;

public class Main {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    String[] line = sc.nextLine().split(" ");

    int a = Integer.parseInt(line[0]);
    int b = Integer.parseInt(line[1]);
    int c = Integer.parseInt(line[2]);
    int count = 0;
    for (int num = a; num <= b; num++) {
      if (c % num == 0) {
        count++;
      }
    }
    System.out.println(count);
  }
}

File: introduction/ITP1_3_D-How Many Divisors?.js
process.stdin.resume();
process.stdin.setEncoding('utf8');
process.stdin.on('data', function (chunk) {
  var line = chunk.toString().split(' ');

  var a = parseInt(line[0]);
  var b = parseInt(line[1]);
  var c = parseInt(line[2]);
  var count = 0;
  for (var num = a; num <= b; num++) {
    if (c % num == 0) {
      count++;
    }
  }

  console.log(count);
});
